### DISCUSSION TITLE
CSV file format

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#525] [Q&A]
Since some insurers appear to be using a CSV file format; what are the expectations for a conformant CSV file?
The issue is with how the nested structures get represented e.g. bundled codes and all of the associated fields.

Without some pre-specified standard we are likely entering the wild west of the hospital price transparency files where each file producer came up with their own format.  The consumers of these files will be back to writing a conversion routine for each file format.

The question about CSV files was [discussed over a year ago ](https://github.com/CMSgov/price-transparency-guide/discussions/58)and it was mentioned that a CSV format could be developed but it wasn't a priority. The insurers seem, or at least one large one, has taken it on themselves, but without the benefit of the GitHub process or an attempt to produce an agreed upon standard mapping from the JSON nested fields to a flat format.

**Comment by shaselton-usds** [comment]
@nmayle Also willing to consider pull requests if you have something in mind.

**Comment by sbenphia** [comment]
I think we will see a variety of formats.  The idea of pulling a portion of each new format and posting it as an example here for comment is probably a start, at least we could begin to identify anything that looked to intentionally obscure the data (a little like the hospital files that were originally hidden from search engines).
